logic.py

```diff
@@ -22,11 +23,13 @@
 
 def production_ready_files(article, file_objects=False):
     """
+    Deprecated.
     Gathers a list of production ready files.
     :param article: an Article object
     :param file_objects: Boolean
     :return: a list of File type objects
     """
+    raise DeprecationWarning('Use get_typesetting_files instead.')
```

Comment:
Just a note: this will mean a major version bump for the typesetting plugin.

Reply:
I only learned this recently but warnings don't cause an exception or terminate the program, so this is what we want to do some time *before* advancing the major version. Removing the function entirely is the change to make when we do a major version bump.

https://docs.python.org/3/library/warnings.html

Reply:
> I only learned this recently but warnings don't cause an exception or terminate the program, so this is what we want to do some time _before_ advancing the major version. Removing the function entirely is the change to make when we do a major version bump.
> 
> https://docs.python.org/3/library/warnings.html

Ace!